modules/edit-modes/src/index.ts

```diff
@@ -79,3 +79,6 @@ export type {
   FeatureCollection,
   AnyGeoJson,
 } from './geojson-types';
+
+// Utils
+export * as utils from './utils';
```

Comment:
Looks like this syntax isn't quite right with the version of TS we're using. The build is failing.

Reply:
Would have helped if I actually tried to build 🤦‍♂️. Pushed fix up/should be resolved now @supersonicclay.